src/components/scenes/TransactionDetailsScene.js

```diff
@@ -63,7 +64,8 @@ type State = {
   direction: string,
   bizId: number,
   category: string,
-  subCategory: string
+  subCategory: string,
+  deviceName: string
```

Comment:
Why is the "deviceName" part of the state for this component. 
This is such a waste of code. Just get it directly on the scene above.